src/main/java/NyanBot.java

```diff
@@ -0,0 +1,237 @@
+            Printer.printEventUsage();
+        } catch (ArrayIndexOutOfBoundsException e) {
+            Printer.printMissingStartEnd();
+            Printer.printEventUsage();
+        }
+    }
+
+    private static String parseCommand(String input) {
+        String[] splitInputs = input.split(" ");
+        return splitInputs[0].toUpperCase();
+    }
+
+    private static void startNyan() {
+        readFile();
+        Printer.printGreet();
+    }
+    private static void getInput() {
+        Scanner scanner = new Scanner(System.in);
+        String[] splitInputs = new String[2];
+
+        while (true) {
+            String input = scanner.nextLine();
+            splitInputs = input.split(" ");
+
+            switch (parseCommand(input)) {
+                case BYE_COMMAND:
+                    scanner.close();
+                    return;
+                case LIST_COMMAND:
+                    System.out.println(LINE);
+                    Printer.printTasks(tasks);
+                    break;
+                case MARK_COMMAND:
+                    int markIndex;
+                    markTask(input);
+                    writeFile();
+                    break;
+                case UNMARK_COMMAND:
+                    unmarkTask(input);
+                    writeFile();
+                    break;
+                case TODO_COMMAND:
+                    addTodo(input);
+                    writeFile();
+                    break;
+                case DEADLINE_COMMAND:
+                    addDeadline(input);
+                    writeFile();
+                    break;
+                case EVENT_COMMAND:
+                    addEvent(input);
+                    writeFile();
+                    break;
+                case HELP_COMMAND:
+                    Printer.printSike();
+                    break;
+                default:
+                    Printer.printInvalidInput();
+                    break;
+            }
+        }
+    }
+
+    private static void readFile() {
+        try {
+            List<String> lines = FileHandler.readFile("data/nyan.txt");
+            for (String line : lines) {
+                tasks.add(readLine(line));
+            }
+        } catch (IOException e) {
+            System.out.println("IO Exception");
+        }
+    }
+
+    private static Task readLine(String line) {
+        String[] tokens = line.split("/");
+        String command = tokens[0].toUpperCase();
+        String status = tokens[1].toUpperCase();
+        Task task = new Task("");
+        switch (command) {
+            case TODO_COMMAND:
+                task = new Todo(tokens[2]);
+                break;
+            case DEADLINE_COMMAND:
+                task = new Deadline(tokens[2], tokens[3]);
+                break;
+            case EVENT_COMMAND:
+                task = new Event(tokens[2], tokens[3], tokens[4]);
+                break;
+            default:
+                return null; //TODO: throw exception
```

Comment:
avoid leaving comments that like notes to self 

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#write-to-the-reader